ledger-core/v2/virtual/object/object.go

```diff
@@ -147,7 +166,7 @@ func (sm *SMObject) stepGetObjectState(ctx smachine.ExecutionContext) smachine.S
 }
 
 func (sm *SMObject) stepWaitState(ctx smachine.ExecutionContext) smachine.StateUpdate {
-	if sm.SharedState.descriptor != nil {
+	if sm.IsReady() {
```

Comment:
Different logic for Inactive, probably? Or is it out of scope?

Reply:
I think it's not in this task

Reply:
+ I think logic in SMObject will be the same.
It will be changed in places, where SMObject is used